pynicotine/gtkgui/widgets/iconnotebook.py

```diff
@@ -141,12 +142,9 @@ def _pack_children(self):
         if sys.platform != "darwin":
             self._add_close_button()
 
-    def _set_text_color(self, color):
+    def _set_text_color(self, color=None):
 
-        color_rgba = Gdk.RGBA()
-
-        if color_rgba.parse(color):
```

Comment:
Someone can manually edit the config to include any value here, so some kind of validation is still desirable.

Reply:
Yes, I'm glad you raised this, we would need our own way of parsing this since we cannot import Gdk into config.

I suppose we need a hexadecimal value in the format #RRGGBB , we could somehow at least check for that even if the color isn't a valid one.